### PR TITLE
feature/home logo

### PR DESCRIPTION
**Pull request by sunilsabatp**
feature->changed the home logo for all tenants except salesforce

### CODE REVIEW

**Comment by norbertschuler** [comment]
I think the new logo does not fit with the line width of the drawings of other menu items.

**Review by norbertschuler — APPROVED** [review]
Technically this pull requests works, but I think HomeLogoSelected does not fit to the design of the other icons.

File: public/assets/images/navigation/HomoLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function HomeLogo() {
```
Comment (mohitb35) [inline public/assets/images/navigation/HomoLogo.tsx:3]: Accept a color prop for HomeLogo

File: src/features/common/Layout/Navbar/getNavBarIcon.tsx [hunk 2]
```diff
@@ -26,9 +34,15 @@ function GetNavBarIcon({ mainKey, router, item }: Props): ReactElement {
...
-        ) : (
+          tenantName === 'salesforce' ? (
+            <GlobeSelected color={themeProperties.primaryColor} />
+          ) : (
+            <HomeLogoSelected color={themeProperties.primaryColor} />
+          )
+        ) : tenantName === 'salesforce' ? (
           <Globe color={themeProperties.light.primaryFontColor} />
+        ) : (
+          <HomeLogo />
```
Comment (mohitb35) [inline src/features/common/Layout/Navbar/getNavBarIcon.tsx:45]: Pass color as a prop for HomeLogo similar to how is done for other icons.

Comment (mohitb35) [inline src/features/common/Layout/Navbar/getNavBarIcon.tsx:45]: Nested ternary conditions are difficult to read later on. Can this be simplified (perhaps extract some of this logic into a separate function).

File: src/features/common/Layout/Navbar/getNavBarIcon.tsx [hunk 3]
```diff
@@ -44,7 +58,13 @@ function GetNavBarIcon({ mainKey, router, item }: Props): ReactElement {
...
       >
         {router.pathname === item.onclick || router.pathname === '/[p]' ? (
-          <DonateSelected color={themeProperties.primaryColor} />
+          tenantName === 'planet' ? (
+            <HomeLogoSelected color={themeProperties.primaryColor} />
+          ) : (
+            <DonateSelected color={themeProperties.primaryColor} />
+          )
+        ) : tenantName === 'planet' ? (
+          <HomeLogo />
```
Comment (mohitb35) [inline src/features/common/Layout/Navbar/getNavBarIcon.tsx:67]: 1. Nested ternary conditions are difficult to read later on. Can this be simplified (perhaps extract some of this logic into a separate function).
2. Pass color as a prop for HomeLogo similar to how is done for other icons.

File: public/assets/images/navigation/HomeLogoSelected.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+interface Prop {
+  color: string;
+}
+function HomeLogoSelected(prop: Prop) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 255.12 255.12"
+      width="24"
```
Comment (mohitb35) [inline public/assets/images/navigation/HomeLogoSelected.tsx:11]: I'd suggest checking out the other navigation icons. They differ in viewbox (most have a height of 512 defined in the viewbox i.e. viewBox="0 0 widthValue 512), and no height/width is explicitly provided. Not defining dimensions within this component itself makes the dimensions easier to manipulate via code (if needed).

File: public/assets/images/navigation/HomoLogo.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 255.12 255.12"
```
Comment (mohitb35) [inline public/assets/images/navigation/HomoLogo.tsx:7]: I'd suggest checking out the other navigation icons. They differ in viewbox (most have a height of 512 defined in the viewbox i.e. viewBox="0 0 widthValue 512), and no height/width is explicitly provided. Not defining dimensions within this component itself makes the dimensions easier to manipulate via code (if needed).

**Review by mohitb35 — CHANGES_REQUESTED** [review]
1. Icons need to be updated to match styles of other items.
2. I think the HomeLogo and HomeLogoSelected icons have been interchanged by mistake.

**Comment by mariahosfeld** [comment]
Looks good to me, asked more stakeholders for their feedback.

**Comment by mariahosfeld** [comment]
This version looks good to me.

File: public/tenants/planet/config.ts [hunk 6]
```diff
@@ -23,15 +23,15 @@
...
-      donate: {
         title: 'home',
         onclick: '/',
         visible: true,
       },
+      // donate: {
+      //   title: 'home',
+      //   onclick: '/',
+      //   visible: true,
+      // },
```
Comment (mohitb35) [inline public/tenants/planet/config.ts:34]: Don't forget to remove this

Comment (mohitb35) [inline public/tenants/planet/config.ts:34]: Don't forget to remove this.

**Review by mohitb35 — APPROVED** [review]

**Pull request merged**